src/libraries/System.Private.CoreLib/src/System/Threading/TimerQueue.Windows.cs

```diff
@@ -12,32 +12,7 @@ private TimerQueue(int id)
             _id = id;
         }
 
-        public static long TickCount64
-        {
-            get
-            {
-                // We need to keep our notion of time synchronized with the calls to SleepEx that drive
-                // the underlying native timer.  In Win8, SleepEx does not count the time the machine spends
-                // sleeping/hibernating.  Environment.TickCount (GetTickCount) *does* count that time,
-                // so we will get out of sync with SleepEx if we use that method.
-                //
-                // So, on Win8, we use QueryUnbiasedInterruptTime instead; this does not count time spent
-                // in sleep/hibernate mode.
```

Comment:
I thought about preserving, but inversing this comment (calling out the potential desync that can occur on Win7). However, we don't support OS prior to Win8/Server 2012 anymore and so such considerations are now largely irrelevant outside of just historical context.

We're now simply matching the behavior to not include "bias" (sleep/hibernation) time that is used on all supported OS and so directly using `Environment.TickCount64` is always fine for us.

Reply:
Right. This internal method is used in ~2 places, so it should not be a problem to include this cleanup in this PR.